index.js

```diff
@@ -74,18 +77,129 @@ function createLeaderBoard(serverScores, message) {
 		let name = member ? member.displayName : 'Member has left the server';
 
 		leaderboard.addField(emoji[i], name, true);
+		leaderboard.addField('With:', breadPoints + ' bread', true);
 		leaderboard.addBlankField(true);
-		leaderboard.addField('With:', breadPoints + ' bread', true)
 	}
 
 	return leaderboard
 };
 
+function wherePlain(serverScores, message) {
```

Comment:
1 function that gets all the data, lets say `getPosition(userIds[])`, use that function in the other functions